src/amo/pages/Guides/index.js

```diff
@@ -0,0 +1,214 @@
+import type { DispatchFunc } from 'core/types/redux';
+import type { ReactRouterMatchType } from 'core/types/router';
+
+import './styles.scss';
+
+type Props = {|
+  match: {
+    ...ReactRouterMatchType,
+    params: {| guidesSlug: string |},
+  },
+|};
+
+type InternalProps = {|
+  ...Props,
+  addons: Array<AddonType>,
+  clientApp: string,
+  dispatch: DispatchFunc,
+  guidesSlug: string,
+  guids: Array<string>,
+  i18n: I18nType,
+  lang: string,
+|};
+
+export const getGuids = (guidesSlug: string): Array<string> | null => {
+  switch (guidesSlug) {
+    case 'privacy':
+      return ['{446900e4-71c2-419f-a6a7-df9c091e268b}'];
+    // The following will be addressed in another issue.
+    // See https://github.com/mozilla/addons-frontend/issues/6744.
+    case 'organize-tabs-and-bookmarks':
+    case 'enhance-your-media-experience':
+    default:
+      return null;
+  }
+};
+
+export const getContent = (guidesSlug: string, i18n: I18nType): Object => {
+  switch (guidesSlug) {
+    case 'privacy':
+      return {
+        title: i18n.gettext('Stay Safe Online'),
+        introText: i18n.gettext(
+          'The web is a wonderful but wild place. Your personal data can be used without your consent, your activities spied upon, and your passwords stolen. Fortunately, extensions can help fortify your online privacy and security.',
+        ),
+        icon: 'stop-hand',
+        sections: [
+          // Bitwarden free password manager
+          {
+            header: i18n.gettext('Create and manage strong passwords'),
+            description: i18n.gettext(
+              'Password managers can help you create secure passwords, store your passwords (safely) in one place, and give you easy access to your login credentials wherever you are.',
+            ),
+            addonCustomText: i18n.gettext(
+              'Fully encrypted password protection. Store your data securely and access logins across devices.',
+            ),
+            exploreMore: i18n.gettext(
+              'Explore more %(linkStart)spassword manager%(linkEnd)s staff picks.',
+            ),
+            exploreUrl: '/collections/mozilla/password-managers/',
+          },
+        ],
+      };
+    // The following will be addressed in another issue.
+    // See https://github.com/mozilla/addons-frontend/issues/6744.
+    case 'organize-tabs-and-bookmarks':
+    case 'enhance-your-media-experience':
+    default:
+      return {};
+  }
+};
+
+export class GuidesBase extends React.Component<InternalProps> {
+  constructor(props: InternalProps) {
+    super(props);
+
+    const { guidesSlug, guids } = this.props;
+
+    this.props.dispatch(
+      fetchGuidesAddons({
+        guids,
+        errorHandlerId: guidesSlug,
+      }),
+    );
+  }
+
+  getGuidesSectionsHtml = (
+    sections: Array<Object>,
+  ): React.ChildrenArray<React.Node> => {
+    const { addons, clientApp, i18n, lang } = this.props;
+
+    return addons.length
+      ? addons.map((addon, key) => {
```

Comment:
What is this `key`? I find it hard to understand how `addons` and `sections` are related.

Reply:
Is it the array index?

Reply:
Each addon that is displayed, is showing its own addon data along with some additional data from the sections array. Since the indexes match up, I am using this to match up the addon with it's corresponding sections data. I am not sure if it will help but I can rename the key to index..?